cmd/brew-bundle.rb

```diff
@@ -1,43 +1,45 @@
 #:  * `bundle` <command>:
 #:    Bundler for non-Ruby dependencies from Homebrew.
 #:
-#:    `bundle` [`install`] [`-v`|`--verbose`] [`--no-upgrade`] [`--file=`<path>|`--global`]:
+#:    `brew bundle` [`install`] [`-v`|`--verbose`] [`--no-upgrade`] [`--file=`<path>|`--global`]:
 #:    Install or upgrade all dependencies in a Brewfile.
 #:
-#:    `brew bundle dump` [`--force`] [`--describe`] [`--file=`<path>|`--global`]
+#:    `brew bundle dump` [`--force`] [`--describe`] [`--file=`<path>|`--global`]:
 #:    Write all installed casks/formulae/taps into a Brewfile.
 #:
-#:    `brew bundle cleanup` [`--force`] [`--zap`] [`--file=`<path>|`--global`]
+#:    `brew bundle cleanup` [`--force`] [`--zap`] [`--file=`<path>|`--global`]:
 #:    Uninstall all dependencies not listed in a Brewfile.
 #:
-#:    `brew bundle check` [`--no-upgrade`] [`--file`=<path>|`--global`] [`--verbose`]
-#:    Check if all dependencies are installed in a Brewfile. Missing dependencies are listed in verbose mode. `check` will exit on the first category missing a dependency unless in verbose mode.
+#:    `brew bundle check` [`--no-upgrade`] [`--file=`<path>|`--global`] [`--verbose`]:
+#:    Check if all dependencies are installed in a Brewfile. Missing dependencies are listed in verbose mode.
+#:    `check` will exit on the first category missing a dependency unless in verbose mode.
 #:
-#:    `brew bundle exec` <command>
+#:    `brew bundle exec` <command>:
 #:    Run an external command in an isolated build environment.
 #:
-#:    `brew bundle list` [`--all`|`--brews`|`--casks`|`--taps`|`--mas`] [`--file=`<path>|`--global`]
+#:    `brew bundle list` [`--all`|`--brews`|`--casks`|`--taps`|`--mas`] [`--file=`<path>|`--global`]:
 #:    List all dependencies present in a Brewfile, optionally limiting by types.
 #:    By default, only brew dependencies are output.
 #:
 #:    If `-v` or `--verbose` are passed, print verbose output.
 #:
-#:    If `--no-upgrade` is passed, don't run `brew upgrade` outdated dependencies.
+#:    If `--no-upgrade` is passed, don't run `brew upgrade` on outdated dependencies.
 #:    Note they may still be upgraded by `brew install` if needed.
 #:
 #:    If `--force` is passed, uninstall dependencies or overwrite an existing Brewfile.
 #:
 #:    If `--zap` is passed, casks will be removed using the `zap` command instead of `uninstall`.
 #:
-#:    If `--file=<path>` is passed, the Brewfile path is set accordingly
+#:    If `--file=`<path> is passed, the Brewfile path is set accordingly.
```

Comment:
I think this should stay as `--file=<path>`

Reply:
This change is because when `brew man` is run, `ronn`'s `--markdown` option converts all angle bracketed `<words>` to `<var>words</var>`, which `man.rb` then converts to backticks. As it's already surrounded by backticks in the source, this causes a backticked word to appear inside a backticked word, resulting in two backticks next to each other, which appear as two literal backticks in the man page's HTML version. There is currently an ugly hack in `man.rb` that prevents this from happening, but the original markup above results in "path" appearing as regular text, instead of monospaced.

Reply:
Thanks for the explanation @EricFromCanada!